components/widgets/inputs/users_emails_input.tsx

```diff
@@ -235,41 +235,74 @@ export default class UsersEmailsInput extends React.PureComponent<Props, State>
         NoOptionsMessage: this.props.suppressNoOptionsMessage ? () => null : this.NoOptionsMessage,
         MultiValueRemove: this.MultiValueRemove,
         IndicatorsContainer: () => null,
+        Input: (props: InputProps) => {
+            const handlePaste = (e: ClipboardEvent) => {
+                if (!this.props.emailInvitationsEnabled) {
+                    return;
```

Comment:
Does this mean we won't be supporting pasting username pasting?

Reply:
I thought we wanted to import bulk emails, not usernames, but re-reading the description, this isn't the case. So I'll refactor to accept usernames as well.